src/Core__Array.resi

```diff
@@ -100,10 +102,11 @@ external setUnsafe: (array<'a>, int, 'a) => unit = "%array_unsafe_set"
 let findIndexOpt: (array<'a>, 'a => bool) => option<int>
 let reverse: array<'a> => array<'a>
 let filterMap: (array<'a>, 'a => option<'b>) => array<'b>
+let keepSome: array<option<'a>> => array<'a>
 let shuffle: array<'a> => array<'a>
 let shuffleInPlace: array<'a> => unit
-let flatMap: (array<'a>, 'a => array<'b>) => array<'b>
-
+@send external flatMap: (array<'a>, 'a => array<'b>) => array<'b> = "flatMap"
```

Comment:
Thanks for binding to the real thing here instead. IIRC I opted to use the last variant because I believe that's what Belt used, but I think the "real" `flatMap` is widely supported now so this should be good.